src/components/Books.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBookAsync } from '../redux/books/booksSlice';
+import BookList from './BooksList';
+import Header from './Header';
+import './Main.css';
+
+// end imports
+
+const App = () => {
+  const dispatch = useDispatch();
+  const [selectedApp] = useState('bMrmeF1Ldn440EWBDcT5');
+
+  const [book, setBook] = useState({
+    title: '',
+    author: '',
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setBook({ ...book, [name]: value });
+  };
+
+  function generateUUID() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+      const r = Math.random() * 16 | 0; const
+        v = c === 'x' ? r : (r & 0x3 | 0x8);
+      return v.toString(16);
+    });
+  }
+
+  const handleAddBook = async () => {
+    const newBook = {
+      itemId: generateUUID(),
```

Comment:
- [ ] when I try to add a book I have a 400 status error which means bad request, this is because in your payload you have `itemId` instead of `item_id` kindly change that
![image](https://github.com/paulshaun-kdtk/React_Book_Store/assets/66097371/26da55d3-1a66-44e1-918a-f8f6a409dd2f)